haystack/document_store/faiss.py

```diff
@@ -164,17 +164,20 @@ def write_documents(self, documents: Union[List[dict], List[Document]], index: O
             )
 
         field_map = self._create_document_field_map()
-        document_objects = [Document.from_dict(d, field_map=field_map) if isinstance(d, dict) else d for d in documents]
+        
+        logger.info("Creating document_objects...")
```

Comment:
Let's not log this message here but instead attach more info to the progress bar ("Creating document objects", the unit etc.) similar to other progress bars in Haystack: https://github.com/deepset-ai/haystack/blob/a7ba1462465d58f317212abb76b035c6a4daac5a/haystack/retriever/dense.py#L243

Please also import `from tqdm.auto import tqdm` and copy the other params from the above code. This will make tqdm usage prettier in jupyter notebooks and some IDEs and avoids new lines for every bar update.